masu/celery/__init__.py

```diff
@@ -23,11 +23,17 @@ def create_celery(app):
     """Create Celery app object using the Flask app's settings."""
     celery = Celery(
         app.import_name,
-        backend=app.config.get('CELERY_RESULT_BACKEND'),
+        # backend=app.config.get('CELERY_RESULT_BACKEND'),
```

Comment:
Any reason to leave this commented out in the code vs. deleted? Or should it not commented out?

Reply:
I suspect at some point, we'll actually want to use the result backend to check task status. So, for now, I'm leaving it there just so whoever comes along to implement that doesn't have to re-wire that stuff up.